addons/account/models/account_move.py

```diff
@@ -1640,10 +1640,10 @@ def _compute_tax_totals_json(self):
                 move.tax_totals_json = None
                 continue
 
-            tax_lines_data = move._prepare_tax_lines_data_for_totals_from_invoice()
+            tax_lines_data, amount_untaxed = move._prepare_tax_lines_data_for_totals_from_invoice()
 
             move.tax_totals_json = json.dumps({
-                **self._get_tax_totals(move.partner_id, tax_lines_data, move.amount_total, move.amount_untaxed, move.currency_id),
+                **self._get_tax_totals(move.partner_id, tax_lines_data, move.amount_total, amount_untaxed, move.currency_id),
```

Comment:
Why do you need to hack into the account module here?  @jjscarafia @Bruno-Zanotti

Reply:
What could be wrong with amount_untaxed on the move? @oco-odoo